src/app/components/dialog/dialog.js

```diff
@@ -28,10 +29,13 @@ export function FooterDialog({
 
     footerEl.style.zIndex = 1;
     return (
-        <dialog className="footer-dialog">
-            <div className="title-bar">
-                <RawHTML Tag="span" html={title} />
-            </div>
+        <dialog className={classes}>
```

Comment:
not a big deal at all, but we typically call `cn` right in the prop, like:
`<dialog className={cn('footer-dialog', className)}>`